src/main/java/marquez/api/mappers/Mapper.java

```diff
@@ -184,7 +184,7 @@ public static JobResponse toJobResponse(@NonNull final Job job) {
         ISO_INSTANT.format(job.getUpdatedAt()),
         job.getInputs().stream().map(input -> input.getValue()).collect(toImmutableList()),
         job.getOutputs().stream().map(output -> output.getValue()).collect(toImmutableList()),
-        job.getLocation().toString(),
+        job.getLocation().map(location -> location.toString()).orElse(null),
```

Comment:
you could use `.map(URL::toString)` here as well

Reply:
Done!